src/include/OpenImageIO/fmath.h

```diff
@@ -137,12 +137,12 @@ pow2rounddown (int x)
 
 
 
-/// Round up to the next whole multiple of m.
-///
-inline int
-round_to_multiple (int x, int m)
+/// Round value up to the next whole multiple.
+/// For example, round_to_multiple(7,10) returns 10.
+template <typename V, typename M>
+inline V round_to_multiple (V value, M multiple)
```

Comment:
Why does `multiple` need its own type? Shouldn't it be `V` as well?

Do we have unit tests for these functions? (in particular to exercise to mixed argument or overflow cases).